modules/openapi-generator/src/main/resources/Java/libraries/jersey3/pom.mustache

```diff
@@ -368,12 +368,15 @@
             <scope>provided</scope>
         </dependency>
         {{/useBeanValidation}}
+        {{#useJakartaAnnotation}}
+        <!-- Jakarta Annotation API support -->
         <dependency>
             <groupId>jakarta.annotation</groupId>
             <artifactId>jakarta.annotation-api</artifactId>
             <version>${jakarta-annotation-version}</version>
             <scope>provided</scope>
         </dependency>
+        {{/useJakartaAnnotation}}
```

Comment:
i generate a jersey2 client using petstore.yaml and removed the above manually and the output compiles without issue (with or without useJakartaEe option enabled)

shall this be simply removed in the first place as the jersey3 client doesn't support the jakarta annotation in the first place?